js/background.js

```diff
@@ -0,0 +1,19 @@
+// context menu
+chrome.contextMenus.create({
+	title: "Add to Are.na",
+	contexts: ["page","selection","link","editable","image","video", "audio"],
+	onclick: function(options){
+		chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+			chrome.tabs.sendMessage(tabs[0].id, {
+				text: "open:dialog",
+				options: options,
+				title: tabs[0].title,
+				url: tabs[0].url
+			}, function(content){});
+		});
+	}
+});
+
+chrome.browserAction.onClicked.addListener(function(tab) {
+	chrome.tabs.executeScript(tab.id, {file: "js/button.js"});
```

Comment:
Ok so yeah, instead of executing a script here, we want to just send the same message as above but pass it slightly different parameters. Like so:

```javascript
chrome.browserAction.onClicked.addListener(function(tab) {
	chrome.tabs.sendMessage(tab.id, {
		text: "open:dialog",
		options: { srcUrl: tab.url },
		title: tab.title,
		url: tab.url
	}, function(content){});
});
````